First Test instance of a class

zahodi opened the pull request:
- gitignore __pycache__ and move bovine.py to lib/

Commit:
gitignore __pycache__ and move bovine.py to lib/

senorsmile reviewed (dismissed):
wouldn't lib/bovine/bovine.py make the module be named:

bovine.bovine

we probably just want 
./bin/bovine.py

so that it is simply call bovine

and importable with

import bovine

zahodi commented:
I will actually rename bovine.py to what it is actually doing.

Commit:
rename bovine.py

Commit:
ignore bovine venv

Commit:
attempt at getting unittest working for static inventory
- add __init__.py files
- import bovine via sys.path
- add requirments.txt for virtualenv
- write first test for staticinventory module

Commit:
rename class

Commit:
test if instance of an object

zahodi commented:
Review Criteria:
- Verify that both Travis checks are green for the latest commit.

senorsmile commented on lib/bovine/staticinventory.py:
```diff
@@ -6,7 +6,7 @@
 
 
-class Inventory:
+class GatherStaticInventory:
```
We will want to rename this.  GatherStaticInventory sounds more like a method name (e.g. VerbObject).  Class names should be a 'class' of things... we will probably want the Class name to either be
- Inventory
of
- StaticInventory

I suppose StaticInventory could be a subclass of Inventory.  

Don't make any changes for now though.  This looks good for initial tests.

senorsmile commented on test/context.py:
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```
this would be way more legible on multiple lines... think FP (functional programming).

senorsmile replied:
Also confirmed all tests passed in travis.  woo hoo!

senorsmile reviewed (approved):
lgtm 👍
